src/functions/util/jsonRequest.js

```diff
@@ -39,12 +37,17 @@ module.exports = async (d) => {
     res = await res.text();
 
     try {
-        data.result = (property?.trim() === '') ? JSON.stringify(JSON.parse(res), null, 2) : eval(`JSON.parse(res)?.${property?.addBrackets()}`);
+        data.result =
+            property?.trim() === ""
+                ? JSON.stringify(res, null, 2)
+                : typeof eval(`JSON.parse(res)?.${property?.addBrackets()}`) === "object" || typeof eval(`JSON.parse(res)?.${property?.addBrackets()}`) === "array"
+                  ? JSON.stringify(eval(`JSON.parse(res)?.${property?.addBrackets()}`), null, 2)
+                  : eval(`JSON.parse(res)?.${property?.addBrackets()}`);
```

Comment:
just use res.json() instead of res.text()

and Array.isArray for checking if it is an array or not

Reply:
> just use res.json() instead of res.text()
> 
> and Array.isArray for checking if it is an array or not

True, will do so.